open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/utils/Constants.java

```diff
@@ -0,0 +1,85 @@
+    public static final String ASSET_SCHEMA_TYPE = "AssetSchemaType";
+    public static final String NAME = "name";
+    public static final String ATTRIBUTE_NAME = "name";
+    public static final String DISPLAY_NAME = "displayName";
+    public static final String AUTHOR = "author";
+    public static final String USAGE = "usage";
+    public static final String SUMMARY = "summary";
+    public static final String ABBREVIATION = "abbreviation";
+    public static final String ENCODING_STANDARD = "encodingStandard";
+    public static final String DATA_TYPE = "dataType";
+    public static final String ELEMENT_POSITION_NAME = "position";
+    public static final String IS_NULLABLE = "isNullable";
+    public static final String IS_UNIQUE = "isUnique";
+    public static final String QUERY = "query";
+    public static final String FORMULA = "formula";
+    public static final String INFO_VIEW_PREFIX = "iv_";
+    public static final String INFORMATION_VIEW = "InformationView";
+    public static final String TYPE_SUFFIX = "_type";
+    public static final String CONNECTION_CONNECTOR_TYPE = "ConnectionConnectorType";
+    public static final String RELATIONAL_COLUMN_TYPE = "RelationalColumnType";
+    public static final String SCHEMA_QUERY_IMPLEMENTATION = "SchemaQueryImplementation";
+    public static final String SEMANTIC_ASSIGNMENT = "SemanticAssignment";
+    public static final String RELATIONAL_TABLE_TYPE = "RelationalTableType";
+    public static final String URL = "url";
+    public static final String EXAMPLES = "examples";
+    public static final String PROTOCOL = "protocol";
+    public static final String ENCRYPTION_METHOD = "encryptionMethod";
+    public static final String NETWORK_ADDRESS = "networkAddress";
+    public static final String ADDITIONAL_PROPERTIES = "additionalProperties";
+    public static final String CONNECTOR_PROVIDER_CLASSNAME = "connectorProviderClassName";
+    public static final String GAIAN_DB_NODE_NAME = "GDName";
+    public static final String USER_ID = "dataPlatformOmas";
+    public static final String CONNECTOR_TYPE = "ConnectorType";
+    public static final String DEPLOYED_DATABASE_SCHEMA = "DeployedDatabaseSchema";
+    public static final String ENDPOINT = "Endpoint";
+    public static final String CONNECTION = "Connection";
+    public static final String SOFTWARE_SERVER = "SoftwareServer";
+    public static final String DATABASE_SERVER = "DatabaseServer";
+    public static final String SERVER_ENDPOINT = "ServerEndpoint";
+    public static final String SOFTWARE_SERVER_CAPABILITY = "SoftwareServerCapability";
+    public static final String ENGINE = "Engine";
+    public static final String REPORTING_ENGINE = "ReportingEngine";
+    public static final String PRIMARY_KEY = "PrimaryKey";
+    public static final String FOREIGN_KEY = "ForeignKey";
+    public static final String DEPLOYED_REPORT = "DeployedReport";
```

Comment:
I think some of the constants are not used and could be removed

Reply:
True. Removed. And also some other constants.